app/views/common/500.html.erb

```diff
@@ -1,21 +1,17 @@
 <%= alert :header => _("Oops, we're sorry but something went wrong"), :class => 'alert-danger',
   :actions => link_to(_('Back'), main_app.root_path, :class => 'btn btn-default'),
-  :text => h(exception.message),
+  :text => h("Error: #{exception_code}, Request ID: #{request_id}"),
   :close => false %>
 
 <div class="pull-right">
   <%= documentation_button("7.3GettingHelp")%>
 </div>
+
 <p id="message">
   <%= _("If you feel this is an error with Foreman itself, please open a new issue with") %>
-  <%= link_to _("Foreman ticketing system"), "http://theforeman.org/projects/foreman/issues", :rel => "external" %>,
-  <%= _("You would probably need to attach the") %>
-  <%= link_to_function(_("Full trace"), "$('#backtrace').toggle()")%>
-  <%= _(", relevant log entries, and it is highly recommended to also attach the foreman-debug output.") %>
+  <%= link_to _("Foreman ticketing system"), "https://projects.theforeman.org/", :rel => "external" %>,
+  <%= _("You would probably need to attach the full trace that can be acquired by command: ") %>
+  <strong> foreman-rake errors:fetch_stacktrace request_id=<%= request_id %></strong>
+  <%= _(" and it is highly recommended to also attach the foreman-debug output.") %>
+  <%= request_id %>
```

Comment:
why do we need the request id again here?

Reply:
Because debug... No, I forgot it to remove :( Thanks!